packages/twenty-docker/prod/.env.example

```diff
@@ -0,0 +1,16 @@
+TAG=latest
+
+POSTGRES_ADMIN_PASSWORD=replace_me_with_a_strong_password
+
+PG_DATABASE_HOST=db:5432
+
+SERVER_URL=http://localhost:3000
+# Uncoment if you are serving your front on another server than the API (eg. bucket)
+# FRONT_BASE_URL=http://localhost:3000
+
+# Use openssl rand -base64 32 for each secret
+# ACCESS_TOKEN_SECRET=replace_me_with_a_random_string_access
```

Comment:
not direcrtly related to this PR but in the 1-liner can we generate those automatically? (maybe you've done it already). also fyi https://github.com/twentyhq/twenty/issues/4587 we'll deprecate them (but not done yet!)